core/common/src/Localization.ts

```diff
@@ -60,13 +60,6 @@ export interface Localization {
    * @throws Error if no keys resolve to a string.
    */
   getLocalizedString(key: string | string[], options?: TranslationOptions): string;
-  /** Similar to `getLocalizedString` but the namespace is a separate param and the key does not include the namespace.
-   * @param namespace - the namespace that identifies the particular localization file that contains the property.
-   * @param key - the key that matches a property in the JSON localization file.
-   * @returns The string corresponding to the first key that resolves.
-   * @throws Error if no keys resolve to a string.
-   */
-  getLocalizedStringWithNamespace(namespace: string, key: string | string[], options?: TranslationOptions): string;
```

Comment:
Isn't this a breaking `@public` API change?

Reply:
gah, I didn't realize that this was public in the interface. It's technically internal in the iTwinLocalization implementation.

I agree with @pmconne, mark as deprecated in this PR is probably the way to go and filing an issue to remove in 4.0.

Reply:
It was easier for me to deprecate in a new PR: https://github.com/iTwin/itwinjs-core/pull/4701. Would it be okay to leave this PR here as a draft for now?

Reply:
Sure.